packages/x-date-pickers-pro/src/internals/hooks/useDesktopRangePicker/useDesktopRangePicker.tsx

```diff
@@ -123,7 +124,7 @@ export const useDesktopRangePicker = <
       timezone,
       autoFocus: autoFocus && !props.open,
       ref: fieldContainerRef,
-      ...(fieldType === 'single-input' && { inputRef }),
+      ...(fieldType === 'single-input' && { inputRef, name }),
```

Comment:
nit: i am wondering why this isn't throwing an error. The problem with the short fused is that if the value is falsey it will get returned. This can happen here, so you are basically trying to spread a boolean value. 🤷🏼

Reply:
Not related to this PR
But if you think the behavior is not safe, we can improve it in a follow up :+1: